No snapshot releases for OSS (#1355)
hazelcast/hz-docs: docs/modules/getting-started/pages/install-hazelcast.adoc

* <<Using the Binary>>
* <<Using Java>>

> To install a slim distribution, append `-slim` to the version number in the commands shown in the following sections; for example, to install the slim distribution of {last-oss-version}, use `{last-oss-version}-slim`.

Reviewer: ```suggestion
To install a slim distribution, append `-slim` to the version number in the commands shown in the following sections; for example, to install the slim distribution of {os-version}, use `{os-version}-slim`.
```
If we go with suggested variables we need to change refs as per updated naming ^

--

To install on macOS, use the Homebrew package manager.

[source,bash,subs="attributes+"]
----
brew tap hazelcast/hz

> brew install hazelcast@{last-oss-version}

Reviewer: ```suggestion
brew install hazelcast@{os-version}
```

. Check that you have the GNU Wget package installed. You'll use Wget to download the GPG keys for the Hazelcast package.
+
[source,bash]
----
wget -V
----
+
If you do not see a version number, download and install link:https://www.gnu.org/software/wget/[Wget^].
+

. Use either of these package managers to install Hazelcast, depending on your Linux distribution:
+
.Debian

[source,shell,subs="attributes+"]
----
wget -qO - https://repository.hazelcast.com/api/gpg/key/public | gpg --dearmor | sudo tee /usr/share/keyrings/hazelcast-archive-keyring.gpg > /dev/null
echo "deb [signed-by=/usr/share/keyrings/hazelcast-archive-keyring.gpg] https://repository.hazelcast.com/debian stable main" | sudo tee -a /etc/apt/sources.list
sudo apt update && sudo apt install hazelcast={last-oss-version}
----
+
.RPM
[source,shell,subs="attributes+"]
----
wget https://repository.hazelcast.com/rpm/stable/hazelcast-rpm-stable.repo -O hazelcast-rpm-stable.repo
sudo mv hazelcast-rpm-stable.repo /etc/yum.repos.d/

> sudo yum install hazelcast-{last-oss-version}

Reviewer: ```suggestion
sudo yum install hazelcast-{os-version}
```

+
--
// tag::maven-full[]

[source,xml,subs="attributes+"]
----
<dependencies>
   <dependency>
       <groupId>com.hazelcast</groupId>
       <artifactId>hazelcast</artifactId>

>        <version>{last-oss-version}</version>

Reviewer: ```suggestion
       <version>{os-version}</version>
```

You can use Hazelcast as a module in the http://openjdk.java.net/projects/jigsaw/[Java Platform Module System] (JPMS).

To run your application with Hazelcast

> libraries on the modulepath, use the `com.hazelcast.core` for `hazelcast-{last-oss-version}.jar` module name.

Reviewer: ```suggestion
libraries on the modulepath, use the `com.hazelcast.core` for `hazelcast-{os-version}.jar` module name.
```

  --add-opens java.management/sun.management=ALL-UNNAMED \
  --add-opens jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED \
  --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED \

>   -jar hazelcast-{last-oss-version}.jar

Reviewer: ```suggestion
  -jar hazelcast-{os-version}.jar
```

  --module-path lib \ <1>
  --module com.hazelcast.core/com.hazelcast.core.server.HazelcastMemberStarter
----

> <1> This example expects the `hazelcast-{last-oss-version}.jar` file in the `lib` directory.

Reviewer: ```suggestion
<1> This example expects the `hazelcast-{os-version}.jar` file in the `lib` directory.
```
